.github/workflows/test.yml

```
@@ -201,3 +201,35 @@ jobs:
           tool: cargo-msrv@0.19
       - name: MSRV Verify
         run: cargo msrv verify
+
+  fuzz:
+    # libFuzzer (and therefore cargo-fuzz) only supports Linux and macOS.
+    if: runner.os != 'Windows'
```

actionlint (1.7.12)

[error] context "runner" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)